zds/pages/views.py

```diff
@@ -79,6 +79,11 @@ def association(request):
     return render(request, "pages/association.html")
 
 
+def accessibilite(request):
+    """Display accessibility's page"""
```

Comment:
Ça c'est de l'anglais bizarre, mieux vaudrait changer pour "Display the page about accessibility".